code-postprocessing/cocopp/pproc.py

```diff
@@ -645,18 +646,19 @@ class DataSet(object):
         testbed_name
         >>> all(ds.evals[:, 0] == ds.target)  # first column of ds.evals is the "target" f-value
         True
-        >>> ds.evals[0::10, (0,5,6)]  # show row 0,10,20,... and of the result columns 0,5,6, index 0 is ftarget
-        array([[  3.98107171e+07,   1.00000000e+00,   1.00000000e+00],
-               [  3.98107171e+05,   2.00000000e+01,   8.40000000e+01],
-               [  3.98107171e+03,   1.61600000e+03,   1.04500000e+03],
-               [  3.98107171e+01,   3.04400000e+03,   3.21000000e+03],
-               [  3.98107171e-01,   4.42400000e+03,   5.11800000e+03],
-               [  3.98107171e-03,   4.73200000e+03,   5.41300000e+03],
-               [  3.98107171e-05,   5.04000000e+03,   5.74800000e+03],
-               [  3.98107171e-07,   5.36200000e+03,   6.07000000e+03]])
-
-        >>> ds.evals[-1,(0,5,6)]  # show last row, same columns
-        array([  1.00000000e-08,   5.67600000e+03,   6.26900000e+03])
+        >>> # investigate row 0,10,20,... and of the result columns 0,5,6, index 0 is ftarget
+        >>> ev = ds.evals[0::10, (0,5,6)]  # doctest:+ELLIPSIS  
+        >>> assert ev[0][0] >= 3.98107170e+07
+        >>> assert ev[0][0] <= 3.98107171e+07
```

Comment:
FTR, you can write several equalities/unequalities in a single line, like 
```python
assert 3.98107170e+07 <= ev[0][0] <= 3.98107171e+07
```

Reply:
I pushed a new version, addressing your comments, @nikohansen. I hope that the branch is now okay to be merged to development.